eogrow/core/area/base.py

```diff
@@ -77,10 +77,10 @@ def get_grid(self, filtered: bool = True) -> Dict[CRS, gpd.GeoDataFrame]:
         grid = self._create_grid()
         self._save_grid(grid, grid_path)
 
-        if filtered and self.config.patch_list is not None:
-            folder_path = self.storage.get_folder(self.config.patch_list.input_folder_key)
-            patch_list_path = fs.path.join(folder_path, self.config.patch_list.filename)
-            relevant_patches = set(load_eopatch_names(self.storage.filesystem, patch_list_path))
+        if filtered and self.config.patch_names is not None:
+            folder_path = self.storage.get_folder(self.config.patch_names.input_folder_key)
+            patch_names_path = fs.path.join(folder_path, self.config.patch_names.filename)
+            relevant_patches = set(load_eopatch_names(self.storage.filesystem, patch_names_path))
```

Comment:
Ok, this is now the only thing left, since it would make me think that `relevant_patches` is of the `List[Tuple[str, BBox]]` type, but is in fact of the `List[str]` type (`Set[str]` really, but still).  So here I would suggest to use any of the following (in that order):

1. `patch_names_subset`
2. `filtered_patch_names`
3. `relevant_patch_names`